lib/r10k/git/rjgit.rb

```diff
@@ -60,6 +60,13 @@ def rev_parse(ver, opts = {})
         return commit.id
       end
 
+      def blob_at(ver, path, opts = {})
+        repo = ::RJGit::Repo.new(opts[:git_dir], is_bare: true)
+        blob = repo.blob(path, ver)
+
+        return blob.data
```

Comment:
Yeah, let me dig down and figure out what hideously long exception classes that can raise....